Signal/src/ViewControllers/ConversationView/ConversationStyle.swift

```diff
@@ -68,7 +40,9 @@ public class ConversationStyle: NSObject {
     @objc public var maxMessageWidth: CGFloat = 0
     @objc public var maxFooterWidth: CGFloat = 0
 
-    @objc public var textInsets = OWSDirectionalEdgeInsets.zero
+    @objc public var textInsetTop: CGFloat = 0
+    @objc public var textInsetBottom: CGFloat = 0
+    @objc public var textInsetHorizontal: CGFloat = 0
```

Comment:
In some contexts, we'll use leading/trailing; in other cases, we'll use left/right.  They're always the same value, so rather than do a bunch of conversions, easiest to use a single value.